src/AzureIoTHub.Portal.Server.Tests.Unit/Pages/DeviceConfigurations/DeviceConfigurationDetailPageTests.cs

```diff
@@ -681,18 +682,20 @@ public void WhenClickToSaveShouldSendPutToTheEndpoint()
                     return true;
                 });
 
```

Comment:
Not part of this PR ?

Reply:
I fixed a unit test that failed randomly on ci. Related to the issue #827